src/main/scala/com/github/unchama/seichiassist/subsystems/mebius/bukkit/listeners/MebiusDropTrialListener.scala

```diff
@@ -35,6 +35,8 @@ class MebiusDropTrialListener[
 
     val player = event.getPlayer
 
+    if (!ManagedWorld.WorldOps(player.getWorld).isSeichi) return
```

Comment:
`ManagedWorld.WorldOps`は`implicit class`
```suggestion
    if (!player.getWorld.isSeichi) return
```